Assets/Scripts/Communication/Communication.cs

```diff
@@ -63,7 +66,7 @@ public static bool Connect(Uri url)
 			if (retry < 5) {
 				Debug.Log("Retrying connection: " + retry);
 				retry++;
-				Thread.Sleep (2000);
+				Thread.Sleep (1000);
```

Comment:
True, it likely isn't especially if we are sleeping during the connect.  I'll take that out.